TA Signals/TempMainView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  TempMainView.swift
+//  TA Signals
+//
+//  Created by Gabriel Maldonado Almendra on 27/7/21.
+//
+
+import SwiftUI
+
+struct TempMainView: View {
+    
+    // 1. The fetch property will observe the FetchToDo class for changes
+    @State var fetchedObject = StocksDataFetcher()
+    @State var showFetchDetails : Bool = false
+    @State var isMarketOpen : Bool = true
```

Comment:
These three booleans seem to never get their state changed.  Is that something you are going to add in later?  If so, I would try to avoid setting them manually.  

isMarketOpen isn't currently being used at all.

Reply:
Yes, is something I still need to add. I'm thinking to use either an `ObservableObject` or a `@published` property wrapped, I'm still thinking how I will approach it. In a nutshell, if the marked is closed it is not needed to fetch new data, or update the UI, or other actions that may happen if the market is open.